support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -52,6 +52,7 @@ const pageQaId = 'qa-paper-subscriptions';
 function PaperLandingPage({
 	productPrices,
 	promotionCopy,
+	participations,
 }: PaperLandingContentPropTypes) {
 	const sanitisedPromoCopy = getPromotionCopy(promotionCopy);
 	const fulfilment: PaperFulfilmentOptions = window.location.pathname.includes(
```

Comment:
an alternative solution might to check the test and swap the fulfilment option here, but not sure if swapping the URL when the page loads is a great idea